gr-iio/python/iio/bindings/iio_types_python.cc

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 Free Software Foundation, Inc.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ */
+
+/***********************************************************************************/
+/* This file is automatically generated using bindtool and can be manually edited  */
+/* The following lines can be configured to regenerate this file during cmake      */
+/* If manual edits are made, the following tags should be modified accordingly.    */
+/* BINDTOOL_GEN_AUTOMATIC(0)                                                       */
+/* BINDTOOL_USE_PYGCCXML(0)                                                        */
+/* BINDTOOL_HEADER_FILE(iio_types.h)                                        */
+/* BINDTOOL_HEADER_FILE_HASH(c42dd62276a1716a7672f12704dabd08)                     */
+/***********************************************************************************/
+
+#include <pybind11/complex.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+namespace py = pybind11;
+
+#include <gnuradio/iio/iio_types.h>
+// pydoc.h is automatically generated in the build directory
+// #include <pluto_sink_pydoc.h>
+
+void bind_iio_types(py::module& m)
+{
+    // ::uhd::stream_cmd_t::stream_mode_t
```

Comment:
Should this comment not be here?

**Adam Horden**

Reply:
good copy pasta catch!